packages/twenty-server/src/engine/core-modules/auth/strategies/google-apis-oauth-common.auth.strategy.ts

```diff
@@ -23,6 +23,7 @@ export class GoogleAPIsOauthCommonStrategy extends PassportStrategy(
       'email',
       'profile',
       'https://www.googleapis.com/auth/gmail.readonly',
+      'https://www.googleapis.com/auth/gmail.send',
```

Comment:
we will need to put that behind a feature flag as we will need to go through the google approval process and it can take up to a month. We will be able to test with a short list of accounts in the meantime and will need the flag :)

Also it's likely that we will need to have a process to ask users to reconnect their account if we detect that the scope is missing (maybe when we try to send an email and get a 403 the first time) We already have similar mechanism for messaging sync

Reply:
yes, is the `IS_ENABLED_WORKFLOW` feature flag enough?